lib/service/service.go

```diff
@@ -4767,13 +4771,13 @@ func (process *TeleportProcess) initProxyEndpoint(conn *Connector) error {
 
 			process.RegisterCriticalFunc("proxy.peer.quic", func() error {
 				if _, err := process.WaitForEvent(process.ExitContext(), ProxyReverseTunnelReady); err != nil {
-					logger.DebugContext(process.ExitContext(), "Process exiting: failed to start QUIC peer proxy service waiting for reverse tunnel server.")
+					logger.DebugContext(process.ExitContext(), "process exiting: failed to start QUIC peer proxy service waiting for reverse tunnel server")
```

Comment:
nit: I think the "agreed" logging style is to capitalize the first word but avoid punctuation.

See the examples in RFD 0154 and this section:

> The message should be a fragment, and not a full sentence. Terminating the message with punctuation should be avoided.

* https://github.com/gravitational/teleport/blob/master/rfd/0154-logging-guidelines.md#logging-standards

Reply:
Word of @rosstimothy is that the message shouldn't be capitalized and "the message should just be another key-value pair" (presumably with "message" as the key), although I agree that the RFD could be clearer on that.

Reply:
We really should update the examples and add a section stating the preferred style, I think very few of us are getting this "right" (me included apparently).